rmw_opensplice_cpp/src/functions.cpp

```diff
@@ -6,7 +6,6 @@
 #include <rmw/allocators.h>
 #include <rmw/error_handling.h>
 #include <rmw/rmw.h>
-#include <rosidl_generator_c/message_type_support.h>
```

Comment:
I just readded this. I also added `#include <rosidl_generator_c/service_type_support.h>` for `rosidl_service_type_support_t`